go/engine/nist_test.go

```diff
@@ -0,0 +1,69 @@
+package engine
+
+import (
+	"github.com/keybase/clockwork"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+	"testing"
+	"time"
+)
+
+func TestNIST(t *testing.T) {
+	tc := SetupEngineTest(t, "nist")
+	defer tc.Cleanup()
+	fu := CreateAndSignupFakeUser(tc, "nst")
+
+	fakeClock := clockwork.NewFakeClockAt(time.Now())
+	tc.G.SetClock(fakeClock)
+	tc.G.ResetLoginState()
+
+	// Need to set active devices
+	Logout(tc)
```

Comment:
how about adding

    nist, err := tc.G.ActiveDevice.NIST(ctx)
    require.Error(t, err, whatever)

to make sure you can't make one of these logged out.